proto/resource_usage_agent.proto

```diff
@@ -17,6 +17,10 @@ service ResourceUsageAgent {
     // Report the CPU time records. By default, the records with the same
     // resource group tag will be batched by minute.
     rpc ReportCPUTime(stream CPUTimeRecord) returns (EmptyResponse) {}
+
+    // Report the summary records. By default, the records with the same
+    // resource group tag will be batched by minute.
+    rpc ReportSummary(stream SummaryRecord) returns (EmptyResponse) {}
```

Comment:
Shall we integrate into one structure? (as they should share the same resource_group_tag)

Reply:
This is possible, but there are some structural inconsistencies between the cpu statistics and the summary statistics in tikv, and some integration work is needed.

Reply:
Could you explain a little bit about what kind of inconsistencies are there? It is not as simple as something I thought previously and I would like to have a deeper learning about the design.

Reply:
CPU statistics usually have a `time window` attribute, and the resource usage related to each tag is bound to the time period (usually one second).
However, the summary statistics are bound to the `request`, and there is no strict time window for the final statistics. (But according to the final demand, maybe the time window still needs to be bound to summary statistics in the end.)
Therefore, the two types of resource usage statistics are only temporarily distinguished in the whole process. If more appropriate, we can also integrate them.

Reply:
Maybe we have some misunderstandings about the scanned lines. According to my understanding, scanned lines are similar to CPU time, recorded for each resourceGroupTag for each second. In UI, when user selects a range of time, user sees the CPU time of each second in that range of time, and the scanned lines of each second in that range of time as well. /cc @crazycs520 Please help verify the behaviour.

Reply:
Oh, it looks like there is a mistake in my understanding. I will relearn the product form today and then make some adjustments to the code :)